uavcan/equipment/gnss/1063.Fix2.uavcan

```diff
@@ -0,0 +1,92 @@
+#
+# GNSS ECEF navigation solution with uncertainty.
```

Comment:
GNSS ECEF **and LLA** navigation solution with uncertainty.

Please mark the old Fix message deprecated (in the header comments).